### DISCUSSION TITLE
Missing `src/data/libraries.ts` file

### DISCUSSION THREAD

**Discussion by chosule** [pmndrs/zustand#2655] [Q&A]
Hello,

I was following the CONTRIBUTING.md guide to help translate the Zustand documentation. However, the guide mentions editing the `src/data/libraries.ts` file in the `pmndrs/website` repository, but I couldn't find this file.

Is there an updated process or an alternative method for contributing to the documentation translation?

Thank you!

**Comment by dbritto-dev** [comment]
@chosule I'm working on revamping docs, so you should wait a little bit

**Reply by dbritto-dev** [reply]
BTW, here's the actual repo which now is legacy -> https://github.com/pmndrs/website-legacy/tree/docs

**Reply by chosule** [reply]
What is the difference between https://github.com/pmndrs/website-legacy/tree/docs and https://github.com/pmndrs/website?

**Reply by chosule** [reply]
@dbritto-dev  and The website-legacy repo is very slow to respond when working with Zustand. Is it okay to translate the documents in the website-legacy repo and then send a PR?

**Reply by dbritto-dev** [reply]
@chosule actually, you don't need to look at website legacy repo since the docs are in this repo under docs folder but keep in mind we are working on revamping docs, so the structure and other things would change
